### PR TITLE
[DX-1462] Operator 0.18.0 Release notes

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

https://tyktech.atlassian.net/browse/DX-1462

---

### Preview Link

<be>

[preview](https://deploy-preview-4910--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-operator/release-notes/operator-0.18/)

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Added comprehensive release notes for Tyk Operator version 0.18, including support lifetime, breaking changes, deprecations, upgrade instructions, release highlights, downloads, and detailed changelog.
- Updated the documentation menu to include a link to the Tyk Operator v0.18 release notes.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>operator-0.18.md</strong><dd><code>Added release notes for Tyk Operator v0.18</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md
<li>Added release notes for Tyk Operator version 0.18.<br> <li> Included sections for support lifetime, breaking changes, <br>deprecations, upgrade instructions, release highlights, downloads, and <br>changelog.<br> <li> Detailed new features and enhancements in the changelog.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4910/files#diff-378ac2678bc3614400d6e8031fcea11ee93050feeb24a5c5db9fdade54871708">+61/-0</a>&nbsp; &nbsp; </td>
</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>menu.yaml</strong><dd><code>Updated menu with Tyk Operator v0.18 release notes</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/data/menu.yaml
- Added menu entry for Tyk Operator v0.18 release notes.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4910/files#diff-a6789c66bfa0660f7fdbe4796002ab4777677bae59a0276689c840100ff0b0b2">+4/-0</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#### Upgrade Instructions
+Go to the [Upgrading Tyk Operator]({{<ref "tyk-stack/tyk-operator/installing-tyk-operator#upgrading-tyk-operator">}}) section for detailed upgrade instructions.
+
+#### Release Highlights
+This release added support for Tyk 5.4 API definition.
+
+For details please refer to the [changelog]({{< ref "#Changelog-v0.18.0">}}) below.
+
+#### Downloads
+- [Docker image v0.18](https://hub.docker.com/r/tykio/tyk-operator/tags?page=&page_size=&ordering=&name=v0.18.0)
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md]:
```suggestion
- [Docker image v0.18.0](https://hub.docker.com/r/tykio/tyk-operator/tags?page=&page_size=&ordering=&name=v0.18.0)
```

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+#### Changelog {#Changelog-v0.18.0}
+
+##### Added
+
+<ul>
+<li>
+<details>
+<summary>Added support of Tyk 5.4 API definition CRD </summary>
+
+Added to ApiDefinition CRD: support of following fields `graphql.proxy.auth_headers` , `graphql.proxy.request_headers`, `graphql.proxy.use_response_extensions`, `graphql.engine.global_headers`.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md]: Expand CRD abbreviation to Custom Resource Definition (CRD) linking to https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/

Link the supported fields to docs, e.g. https://tyk.io/docs/universal-data-graph/concepts/header_management/#global-headers-for-udg

**Review by dcs3spp — CHANGES_REQUESTED** [review]
PR LGTM @caroltyk I have reviewed with minor suggestions of expanding the CRD abbreviation and linking to docs pages for supported GraphQL fields.

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md (hunk 2 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md]: thanks. docs updated

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#### Changelog {#Changelog-v0.18.0}
+
+##### Added
+
+<ul>
+<li>
+<details>
+<summary>Added support of Tyk 5.4 API definition CRD </summary>
+
+Added to ApiDefinition [Custom Resource Definition (CRD)](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/): support of following fields [GraphQL APIs headers]({{<ref "graphql/gql-headers">}}) `graphql.proxy.auth_headers` , `graphql.proxy.request_headers`, `graphql.proxy.use_response_extensions`, [UDG Header management]({{< ref "universal-data-graph/concepts/header_management" >}}) `graphql.engine.global_headers`.
```
Comment (komalsukhani) [inline tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md]: Following fields were added in Operator

- `introspection` option to enable/disable GraphQL introspection
- features section to graphql proxy config
- AuthHeaders
- SubscriptionType
- RequestHeaders
- UseResponseExtensions 
- RequestHeadersRewrite

**Review by komalsukhani — CHANGES_REQUESTED** [review]

**Comment by dcs3spp** [comment]
@caroltyk Once the [request changes](https://github.com/TykTechnologies/tyk-docs/pull/4910#discussion_r1656504281) have been approved by @komalsukhani I will merge ready for release.

File: tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md (hunk 3 above)
Comment (carolwkc) [reply tyk-docs/content/product-stack/tyk-operator/release-notes/operator-0.18.md]: updated

**Review by komalsukhani — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**